src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -56,23 +55,18 @@ def execute(self, context: Any) -> Table:
         if self.file_conn_id:
             BaseHook.get_connection(self.file_conn_id)
 
+        database = create_database(self.output_table.conn_id)
         self.normalize_config = populate_normalize_config(
             ndjson_normalize_sep=self.ndjson_normalize_sep,
-            database=create_database_from_conn_id(self.output_table.conn_id),
+            database=database,
         )
 
-        hook = get_hook(
-            conn_id=self.output_table.conn_id,
-            database=self.output_table.metadata.database,
-            schema=self.output_table.metadata.schema,
-            warehouse=self.output_table.metadata.warehouse,
+        return self.load_data(
```

Comment:
Perhaps we could call `database.load_pandas_dataframe_to_table` from here, without the need to have a `load_data` method1

Reply:
We have to handle the patterns in path, that is done in load_data.